### PR TITLE
maven-completion 20160501

### PR DESCRIPTION
**Pull request by mnuessler**

### CODE REVIEW

File: maven-completion.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@ class MavenCompletion < Formula
   desc "Bash completion for Maven"
   homepage "https://github.com/juven/maven-bash-completion"
   url "https://github.com/juven/maven-bash-completion.git",
-    :revision => "e27e290a8aa3e09eff08792df27f0e1fe230fbd6"
-  version "20160114"
+    :revision => "106b6ca03badc9474cf7b6b1b7039ad950a17f89"
+  version "20160501"
```
Comment (bfontaine) [inline maven-completion.rb]: We should ask upstream for tagged releases instead of using arbitrary versions here.

Comment (mnuessler) [inline maven-completion.rb]: I can open a ticket and ask for tagged releases. However, would `brew upgrade` still work if we would switch to version numbers such as `x.y.z`?

Comment (bfontaine) [inline maven-completion.rb]: No it won’t; good catch! We’ll need to hack around this by using e.g. `YYYYMMDD.x.y.z`.

**Comment by bfontaine** [comment]
Let’s merge this while we wait for upstream. Thank you @mnuessler! 🎉

**Comment by mnuessler** [comment]
@bfontaine Thanks!

**Pull request closed without merging**